### PR TITLE
Reduce the number of references to JMXFetch version

### PR DESCRIPTION
**Pull request by albertvaka**
### What does this PR do?

Make omnibus rename the file so it doesn't include the version in its name.

### Motivation

This reduces the number of files we need to modify when bumping the JMXFetch version from 3 to 1.

### CODE REVIEW

**Comment by albertvaka** [comment]
This other change is related: https://github.com/DataDog/jmxfetch/pull/218

So we don't lose info about the version info installed.

File: omnibus/config/software/jmxfetch.rb [hunk 1]
```diff
@@ -26,5 +25,8 @@
   ship_license "https://raw.githubusercontent.com/DataDog/jmxfetch/master/LICENSE"
   mkdir jar_dir
   copy "jmxfetch-#{jmxfetch_version}-jar-with-dependencies.jar", jar_dir
-  block { File.chmod(0644, "#{jar_dir}/jmxfetch-#{jmxfetch_version}-jar-with-dependencies.jar") }
+  block do
+    File.rename("#{jar_dir}/jmxfetch-#{jmxfetch_version}-jar-with-dependencies.jar", "#{jar_dir}/jmxfetch.jar")
```
Comment (arbll) [inline omnibus/config/software/jmxfetch.rb]: nit: we can probably achieve the same result by changing the target path of the `copy` above.

**Review by arbll — COMMENTED** [review]
LGTM, just a small nit

File: omnibus/config/software/jmxfetch.rb (hunk 1 above)
Comment (albertvaka) [reply omnibus/config/software/jmxfetch.rb]: Indeed, it works with the copy.

**Review by arbll — APPROVED** [review]

**Pull request merged**